cockroachdb/tests/docker/docker-compose.yaml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+
+services:
+  cockroachdb:
+    container_name: cockroachdb
+    image: cockroachdb/cockroach:latest
```

Comment:
do we want this to be latest? I think we should pin this to versions, if they update and it breaks the tests it could be bad.

Reply:
I think that's a good thing, we should always be supporting the latest (official) version, right?

Reply:
We should pin some versions because otherwise we could be breaking the check against prior versions of the database as well